modules/ballerina-logging/src/main/java/org/ballerinalang/logging/BLogManager.java

```diff
@@ -54,8 +54,8 @@
 
     @Override
     public void readConfiguration(InputStream ins) throws IOException, SecurityException {
-        Properties properties = new Properties();
-        properties.load(ins);
+        Properties properties = getDefaultLogConfiguration();
+        properties.load(ins); // override the default configs using the provided config file
```

Comment:
What if someone has not provided a config file?
IMO we should first check whether a config file has been provided. Then get the default configs only if a custom one is not provided.

Reply:
Ack. Will make the necessary ammendments